WORKSPACE

```diff
@@ -29,7 +29,7 @@ http_archive(
 )
 
 ##### GRPC
-_GRPC_VERSION = "1.36.1"
```

Comment:
I don't think there's any need to change this; stick with the version that they are using in master/main.